spec/rails_helper.rb

```diff
@@ -70,4 +71,9 @@
   # config.filter_gems_from_backtrace("gem name")
 
   config.include ActiveSupport::Testing::TimeHelpers
+
+  config.include FactoryBot::Syntax::Methods
+  config.before(:suite) do
+    FactoryBot.reload
```

Comment:
what does this do?

Reply:
Without this line FactoryBot was complaining when I ran the specs. Like it didn't properly load all the factories. I could've also had something else configured wrong or my machine was in a weird state